db/migrate/20211020062413_create_question_answer_votes.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class CreateQuestionAnswerVotes < ActiveRecord::Migration[6.1]
+  def up
+    create_table :question_answer_votes do |t|
+      t.integer :post_id, null: false
+      t.integer :user_id, null: false
+      t.datetime :created_at, null: false
+    end
+
+    add_index :question_answer_votes, [:post_id, :user_id], unique: true
+
+    execute <<~SQL
+    INSERT INTO question_answer_votes (post_id, user_id, created_at)
+    SELECT
+      X.post_id AS post_id,
+      (X.value->>'user_id')::int AS user_id,
+      (X.value->>'created_at')::timestamp AS created_at
+    FROM (
+      SELECT
+        post_id,
+        jsonb_array_elements(value::jsonb) AS value
+      FROM post_custom_fields WHERE name = 'vote_history'
+    ) AS X
+    WHERE (X.value->>'action') != 'destroy'
+    ORDER BY (X.value->>'created_at')::timestamp DESC
+    ON CONFLICT DO NOTHING
```

Comment:
We've decided to not support multiple votes per user. Too complicated for V0 for us.